src/DashboardUI/src/pages/application/create/ApplicationCreatePage.tsx

```diff
@@ -592,11 +684,26 @@ function ApplicationCreatePageForm() {
         </ApplicationFormSection>
 
         <ApplicationFormSection title="Supporting Documents (Optional)">
+          {showUploadDocumentError && (
```

Comment:
```suggestion
          {showUploadDocumentError && uploadDocumentErrorMessage && (
```
just in case - check if we have a message as well so we never show an empty alert box

Reply:
Does it make sense to make these the same? If there's an error we set the message? If there's not we set the message to null?